packages/pds/src/api/com/atproto/server/confirmEmail.ts

```diff
@@ -9,7 +9,10 @@ export default function (server: Server, ctx: AppContext) {
     handler: async ({ auth, input, req }) => {
       const did = auth.credentials.did
 
-      const user = await ctx.accountManager.getAccount(did)
+      const user = await ctx.accountManager.getAccount(did, {
+        includeDeactivated: true,
+        includeTakenDown: true,
```

Comment:
I don't think we let takendown accounts to this point thanks to `accessCheckTakedown`.

Reply:
ah right 👍